src/lib/Scenes/Home/Components/ForYou/index.tsx

```diff
@@ -12,17 +12,13 @@ interface DataSourceRow {
   data: any
 }
 
-interface Props extends ViewProperties, RelayProps {
-  trigger1pxScrollHack?: boolean
-}
-
 interface State {
   modules: any[]
   isRefreshing: boolean
   dataSource: ListViewDataSource
 }
 
-export class ForYou extends React.Component<Props, State> {
+export class ForYou extends React.Component<any, State> {
```

Comment:
Only the scroll hack prop needs to be removed, this component still needs those `ViewProperties` and `RelayProps`.